Source/FieldSolver/FiniteDifferenceSolver/EvolveB.cpp

```diff
@@ -355,9 +364,15 @@ void FiniteDifferenceSolver::EvolveBCartesianECT (
                         jp = j + vec(0);
```

Comment:
At the beginning of `FiniteDifferenceSolver::EvolveBCartesianECT`:
Let's also add a check here that aborts if this is not a 2D or 3D simulation.